geode-core/src/main/java/org/apache/geode/cache/asyncqueue/internal/InternalAsyncEventQueue.java

```diff
@@ -28,4 +28,6 @@
   void stop();
 
   void destroy();
+
+  boolean isPartitionedRegionClearUnsupported();
 }
```

Comment:
I added it to the interface so PartitionedRegionClear.requireAsyncEventQueuesSupportPartitionedRegionClear() can depend on just the interface rather than the concrete classes.